src/pages/SignupPage/SignupPage.jsx

```diff
@@ -15,12 +18,31 @@ function SignupPage() {
     });
   }
 
+  const userData = JSON.parse(localStorage.getItem('users')) || [];
+
+  function signUp() {
+    if (formData.password == formData.confirmPassword) {
+      localStorage.setItem('users', JSON.stringify([...userData , {email: formData.email, password:formData.password}]));
```

Comment:
- Yönlendirmeler için react-router-dom'un useNavigate'i  import edilecek.
`import { Link, useNavigate } from "react-router-dom";`
- State'lerin altında useNavigate kullanılacak.
`  const navigate = useNavigate();`
- Ardından istenilen kod bloğunda bu şekilde navigate yapılabilir
`navigate("/login", { replace: true });`